sdk/wallet-adapter/wallet-kit-core/src/index.ts

```diff
@@ -192,6 +193,19 @@ export function createWalletKitCore({
       };
     },
 
+    selectAccount(account) {
+      if (
+        account === internalState.currentAccount ||
```

Comment:
n00b ques alert: is `internalState` non nullable?

Reply:
Yeah it's initialized to an object and isn't redefined